tests/terraform_setup/fargate/terraform.tf

```diff
@@ -28,7 +25,7 @@ locals {
   aws_ecs_service_stack_name = "${var.aws_resource_prefix}-svc-stack"
 
   # The name of the ECR repository to be created
-  aws_ecr_repository_name = var.aws_resource_prefix
+  aws_ecr_repository_name = "${var.aws_resource_prefix}"
```

Comment:
Just like before, there is no need to use a string interpolation here, is there?

Reply:
No need, I had updated the name of the resources and let the string interpolation syntax in case it was required again.